### PR TITLE
Build static dependencies

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: projects/apple/f3d.bundle.cmake [hunk 1]
```diff
@@ -24,7 +24,7 @@ if (ospray_enabled)
   endforeach ()
 endif ()
 
-# Bundle F3D and all binaries
+# Bundle F3D and all binaries. No plugins on macOS.
```
Comment (mwestphal) [inline projects/apple/f3d.bundle.cmake:27]: remove comment, no plugins to bundle anywhere yet

File: projects/f3d.cmake [hunk 2]
```diff
@@ -38,11 +41,31 @@ superbuild_add_project(f3d
...
+    DEPENDEES
+      install
+  )
+endif ()
+
+if (BUILD_SHARED_LIBS_f3d)
+  superbuild_project_add_step("extra-install"
+    COMMAND ${CMAKE_COMMAND} --install <BINARY_DIR> --component mimetypes
+    COMMAND ${CMAKE_COMMAND} --install <BINARY_DIR> --component configuration
+    COMMAND ${CMAKE_COMMAND} --install <BINARY_DIR> --component sdk
```
Comment (mwestphal) [inline projects/f3d.cmake:65]: should only be sdk

File: projects/f3d.cmake [hunk 3]
```diff
@@ -37,12 +40,30 @@ superbuild_add_project(f3d
...
   COMMAND ${CMAKE_COMMAND} --install <BINARY_DIR> --component configuration
-  COMMAND ${CMAKE_COMMAND} --install <BINARY_DIR> --component sdk
   COMMENT
-    "Installing extra components"
+    "Installing configurations"
   DEPENDEES
     install
 )
+
+if (UNIX)
```
Comment (Meakk) [inline projects/f3d.cmake:51]: And not Apple?

File: .github/actions/f3d-superbuild/action.yml [hunk 4]
```diff
@@ -75,16 +75,21 @@ runs:
...
-      -Df3d_GIT_TAG=${{ inputs.f3d_version }}
-      -Dvtk_SOURCE_SELECTION=git
-      -Dvtk_GIT_TAG=${{ env.VTK_COMMIT_SHA }}
-      -DUSE_SYSTEM_python3=ON
-      -DCMAKE_C_COMPILER_LAUNCHER=sccache
       -DCMAKE_CXX_COMPILER_LAUNCHER=sccache
+      -DCMAKE_C_COMPILER_LAUNCHER=sccache
       -DENABLE_ospray=${{ inputs.raytracing_label == 'raytracing' && 'ON' || 'OFF' }}
       -DOCCT_ENABLE_COLORING=${{ runner.os == 'macOS' && 'OFF' || 'ON' }}
+      -DUSE_SYSTEM_python3=ON
```
[the remark this answers is not in the document]
Comment (mwestphal) [reply .github/actions/f3d-superbuild/action.yml:88]: its not even enabled, to be removed

File: projects/assimp.cmake [hunk 5]
```diff
@@ -1,19 +1,21 @@
...
+    -DASSIMP_BUILD_ASSIMP_TOOLS=OFF
     -DASSIMP_BUILD_COLLADA_IMPORTER=ON
     -DASSIMP_BUILD_DXF_IMPORTER=ON
-    -DASSIMP_BUILD_TESTS=OFF
-    -DASSIMP_BUILD_ASSIMP_TOOLS=OFF
+    -DASSIMP_BUILD_FBX_IMPORTER=ON
+    -DASSIMP_BUILD_OFF_IMPORTER=ON
     -DASSIMP_BUILD_SAMPLES=OFF
+    -DASSIMP_BUILD_TESTS=OFF
+    -DASSIMP_BUILD_ZLIB=ON
```
[the remark this answers is not in the document]
Comment (mwestphal) [reply projects/assimp.cmake:14]: i'd rather not, as zlib license conflicts with packaged zlib when installing the package. This is a common superbuild issue but I dont have a good idea to fix it

[the remark this answers is not in the document]
Comment (Meakk) [reply projects/assimp.cmake:14]: Ok

**Review by Meakk — APPROVED** [review]

**Pull request merged**